config/config-mysql_apache_jvm.yaml

```diff
@@ -0,0 +1,83 @@
+# This config hopes to mimick the Cloud Monitoring Agent by collecting metrics from popular third-party services the Agent already supports
+# and formatting the metrics in the same way, so they appear similarly in Cloud Monitoring. The services targeted here are MySQL, Apache and JVM.
+# Please remove/comment-out the prometheus_exec configs that you don't need in the receivers section as well as the list in the service/pipelines/metrics/receivers section.
+# Documentation specific to each exporter is found in the comment directly above its configuration
+
+receivers:
+
+  # https://github.com/prometheus/mysqld_exporter
+  prometheus_exec/mysql:
+    exec: ./prometheus_exporters/mysqld_exporter --web.listen-address=":{{port}}" --no-collect.global_variables  # this last flag reduces the amount of generated metrics, can be removed to enable all (600+) MySQL metrics
```

Comment:
I'd move that comment to the line above (likewise elsewhere in this config file where you have long trailing comments)